content/pages/getting-started.md

```diff
@@ -0,0 +1,72 @@
+Title: Getting Started
+Slug: getting-started
+Summary: Getting started implementing the SpaceAPI for your hackerspace.
+SortOrder: 20
+
+If you want to implement the SpaceAPI for your hackerspace you need to do the
+following:
+
+ - [ ] Host the JSON file and a logo somewhere
+ - [ ] Create a
+   [pull-request](https://help.github.com/en/articles/creating-a-pull-request)
+   for the [directory repository](https://github.com/spaceapi/directory) to add
+   your space to the directory.
+
+# Hosting
+
+The most minimal solution is to just [statically
+host](https://en.wikipedia.org/wiki/Static_web_page) the JSON file on your own
+server.
+
+A most minimal JSON file with just the required fields looks like that:
+
+    :::json
```

Comment:
This syntax doesn't seem to be supported by GitHub.

Reply:
Well I just used the same as on the [home.md page](https://github.com/SpaceApi/website/blob/master/content/pages/home.md). But we can change it to GitHub flavored / common mark syntax if pelican supports that as well.

Reply:
Ah, I forgot that this is rendered by Pelican and not by Github. In that case, leave it :slightly_smiling_face:

Reply:
GitHub flavored markdown isn't recognized by pelican. But we could just remove the `:::json` since it doesn't have highlighting anyways.

Reply:
You could also use :::js (or :::javascript?), that should get parsed by Pelican.